CONTRIBUTING.md

```diff
@@ -1,61 +1,22 @@
-# How to contribute
+## Update from upstream
```

Comment:
Thank you for making this file usable. I'm always looking for instructions like these in `CONTRIBUTING.md`.